lib/default_host.rb

```diff
@@ -0,0 +1,13 @@
+class DefaultHost
+  def call(request)
+    raise "ENV['DOMAIN_NAME'] cannot be blank" if default_host.blank?
```

Comment:
Maybe get this to blow up earlier, like in `config/initializers/figaro.rb` instead of the first request.

Reply:
Like in AssetHosts, this is to protect against an empty string. I will change `.blank?` to `.empty?` and change the error message to `cannot be an empty string`.

Reply:
Alternatively, I can write a ConfigValidator initializer class that will make sure these are not set to empty strings. That way, it won't perform this check on every request.

Reply:
👍